src/views/players/player/player_misc.cpp

```diff
@@ -31,6 +32,16 @@ namespace big
 		ImGui::Checkbox("NEVER_WANTED"_T.data(), &g_player_service->get_selected()->never_wanted);
 		ImGui::Checkbox("SEMI_GODMODE"_T.data(), &g_player_service->get_selected()->semi_godmode);
 		ImGui::Checkbox("VIEW_NET_SESSION_FIX_VEHICLE"_T.data(), &g_player_service->get_selected()->fix_vehicle);
+
+		components::button("Trigger UFO Abduction", [] {
```

Comment:
This needs a translation label, no more hardcoded text.

Reply:
Why is this added in the first place? There are more appropriate methods to achieve this in Lua. Additionally, we previously removed the Gooch feature, so adding this now doesn't make sense.

Reply:
Since normal users can't comment anything anymore the only way for me is by this section, "Explode Player" doesn't work!!!, crashes the game, i wish you guys opened comments to everyone so we can give feedback about crashes (btw i'm using the [f3e975f] build

Reply:
"Explode Player" crashes the game (latest "update" branch build)
![image](https://github.com/YimMenu/YimMenu/assets/97450182/a502aa18-2194-46ea-ae3b-d0654c739790)

Reply:
x64-Debug log
![image](https://github.com/YimMenu/YimMenu/assets/97450182/31630c85-b901-4ba1-8243-9f087ac76e66)

Reply:
> `explosion_anti_cheat_bypass::m_set_script_flag = memory::byte_patch::make(g_pointers->m_gta.m_blame_explode.sub(0x63).as<uint32_t*>(), 0x90909090).get();`

That fixed it :D, thanks!
![image](https://github.com/YimMenu/YimMenu/assets/97450182/d1df526e-db8e-4b27-9cb3-256174e82506)